README.md

```diff
@@ -85,9 +86,9 @@ The Puppet Extension for VSCode works with Puppet 4 or higher. Some features wil
 
 ## Requirements
 
-You will need to have the [Puppet Agent](https://puppet.com/docs/puppet/4.10/about_agent.html) or [Puppet Development Kit (PDK)](https://puppet.com/docs/pdk/1.x/pdk.html) installed in order to fully use this extension.
+You will need to have the [Puppet Development Kit (PDK)](https://puppet.com/docs/pdk/1.x/pdk.html) or [Puppet Agent](https://puppet.com/docs/puppet/4.10/about_agent.html) installed in order to fully use this extension.
```

Comment:
Should probably use latest instead of EOL puppet 4

`https://puppet.com/docs/puppet/latest/about_agent.html`